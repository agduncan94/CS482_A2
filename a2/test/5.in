
>seq1

AAD

DLM

F

>seq2

MOMOSFOSFMSFAADDFMIFEFEFEF
