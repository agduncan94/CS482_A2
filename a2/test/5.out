4
AADDLM-F
AADDFMIF
